src/features/discussions/DiscussionDropDown.tsx

```
@@ -29,8 +33,19 @@ export default function DiscussionDropDown({ ...props }: IDiscussionDropDownProp
               </li>
             </>
           ) : (
-            <li className="py-2 px-3 hover:bg-primary/70 rounded-xl">신고하기</li>
+            <li
+              className="py-2 px-3 hover:bg-primary/70 rounded-xl"
+              onClick={() => setIsOpenReportModal(true)}
+            >
+              신고하기
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)